src/hooks/useHashTagsForm.ts

```diff
@@ -0,0 +1,26 @@
+import { IHashTagsForm } from '@/interfaces/formInterfaces';
+import { useMemo } from 'react';
+import { useForm } from 'react-hook-form';
+
+const useHashTagsValues = () => {
```

Comment:
I don't understand this how it is supposed to work. It is a hook which never changes the value of formValues

Reply:
This form was created into its own function, because all other forms have also been sepparated into their own functions. Since react-hook-form is a hook, creating a sepparate function for retrieving the form values also needs to be a hook. That's the only reason.

Reply:
alright